installer/functions/task-router/deploy-task-router.ts

```diff
@@ -0,0 +1,124 @@
+// Imports global types
+import '@twilio-labs/serverless-runtime-types';
+import { createTaskQueue, createWorkspace, getAllWorkflows, getAllTaskQueues, createWorkflow, QueueToQueueSid, getAllWorkers } from './task-router-helpers.private';
+import { 
+  WORKSPACE_FRIENDLY_NAME,
+  SCHEDULERS,
+  EDUCATORS,
+  SCHEDULING,
+  EDUCATION,
+  INTAKE_BY_SCHEDULERS,
+  TRANSFER_TO_NURSE_EDUCATOR
+} from './task-router-helpers.private';
+// Fetches specific types
+import {
+  Context,
+  ServerlessCallback,
+  ServerlessFunctionSignature,
+} from '@twilio-labs/serverless-runtime-types/types';
+import { WorkflowInstance } from 'twilio/lib/rest/taskrouter/v1/workspace/workflow';
+
+export const handler: ServerlessFunctionSignature = async function(
```

Comment:
this is a really long function; it might be a good practice to break this up into smaller, composable functions that do one thing only